# Topics
topic_route: "route"
topic_map: "/vision_cone_detector/estimated_map/"
topic_pursuit: "pursuit_point"
topic_state: "state"
topic_delaunay: "/delaunay_detector/simplices"
